openage/convert/service/export/interface/visgrep.pyx

```diff
@@ -302,7 +302,8 @@ cdef visgrep_cli(geom_params, metric_params, img_params):
     """
 
     cdef image_t img, find
-    cdef vector[image_t] matches
+    # cdef vector[pixel_t[:, :, :]] matches
+    cdef list matches
```

Comment:
vector would be more performant, doesn't that work any more?

Reply:
```
Error compiling Cython file:
------------------------------------------------------------
...
    """
    Perform search, return list of results.
    """

    cdef image_t img, find
    cdef vector[image_t] matches
               ^
------------------------------------------------------------

openage/convert/service/export/interface/visgrep.pyx:305:15: Reference-counted type 'image_t' cannot be used as a template argument
```

it fails for the tenplate argument. maybe they changed something about memoryviews in cython 3 but I couldn't find anything in the migration docs.

Reply:
but image_t is not a memoryview, is it? it just used a cpp vector, but well lets use a py list, the performance doesn't matter that much here

Reply:
`image_t` is declared as 

```
ctypedef numpy.uint8_t pixel_t
ctypedef pixel_t[:, :, :] image_t
```

so it is a memoryview on a `uint8_t`. Is there a reason why you used it like this?

Reply:
see https://github.com/cython/cython/issues/3085 where it got removed